chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java

```diff
@@ -90,7 +90,10 @@ public void writeModelsToDisk() {
         chunker.save();
         System.out.println("Done training, models are in " + rm.getString("modelDirPath"));
     }
-
+    public void writeModelsToDisk(String dir, String modelName){
+        chunker.write(dir + modelName + ".lc", dir + modelName + ".lex");
```

Comment:
Semaphore is angry. Invited you to Semaphore account. Go check it out and see why it's failing.